src/client/job.rs

```diff
@@ -202,12 +204,23 @@ pub fn print_job_detail(
             .cell(),
     ]);
 
+    rows.push(vec![
+        "Submission date".cell().bold(true),
+        job.submission_date.round_subsecs(0).cell(),
+    ]);
+
+    rows.push(vec![
+        "Makespan".cell().bold(true),
```

Comment:
Maybe Duration instead of Makespan?

Reply:
I want to reserve "duration" for time that is actively processed. Makespan is from submission to completion, counting even time when there were no workers at all or tasks of job was suppresed by a job with a higher prioirty.